ScoutSuite/providers/aws/configs/regions.py

```diff
@@ -73,7 +76,7 @@ def __init__(self, service_metadata=None, thread_config=4):
                     continue
                 params = resource_metadata['params'] if 'params' in resource_metadata else {}
                 ignore_exceptions = True if 'no_exceptions' in resource_metadata and \
```

Comment:
```suggestion
                 ignore_exceptions = resource_metadata.get('no_exceptions', False)
```

Reply:
Also, make sure that not just the first line is replaced, I didn't mange to select the whole statement at once.